cpp/lib/src/MARC.cc

```diff
@@ -2937,10 +2937,42 @@ std::set<std::string> ExtractPrintCrossLinkPPNs(const MARC::Record &record) {
 }
 
 
-std::set<std::string> ExtractPrintAndOnlineCrossLinkPPNs(const MARC::Record &record) {
+std::set<std::string> ExtractOtherCrossLinkPPNs(const MARC::Record &record) {
+    std::set<std::string> cross_reference_ppns;
+    for (const auto _785_field : record.getTagRange("785")) {
+        const auto ppn(BSZUtil::GetK10PlusPPNFromSubfield(_785_field, 'w'));
+        if (ppn.empty())
+            continue;
+
+        const auto subfield_i_contents(_785_field.getFirstSubfieldWithCode('i'));
+        if (subfield_i_contents == "Forts." or subfield_i_contents == "Fortgesetzt durch") {
+            cross_reference_ppns.emplace(ppn);
+            continue;
```

Comment:
I think this continue is not necessary, since there are no other statements after it until the end of the loop